PDR-0008-Maintainer-Committee/README.md

```diff
@@ -0,0 +1,52 @@
+---
+PDR: ?
+Title: Maintainer Committees
+Category: Guidelines
+Status: Pending
+Authors:
+    - Christina Gianelloni <christina@blinklabs.io> 
+Created: 2025-02-16
+---
+
+## Context
+The [Statutes](https://github.com/pragma-org/PDRs/blob/main/.incorporation/20240422_Attachment_Statutes_Statuten_PRAGMA_signed.pdf) clearly call out the need for Maintainer Committees and Leads therein. However that information is spread throughout the statutes and at times leaves room for ambiguity. They organize, manage and implement a software project and will need guidance on the expectations and requirements within PRAGMA. 
+
+## Motivation
+Clearly defining the structure and responsibilities of the Maintainer Committees and their Leads are essential for accountability, transparency and efficiency.
+
+## Decision
+Maintainer Committees
+- Composition: The Maintainer Committee comprises individuals who have demonstrated consistent contributions to a PRAGMA project. The Lead appoints the other members. Thereafter members are appointed by invitation, from an existing member, based on merit and commitment.
+- Structure: They report to the Administrative Board in phase 1. In phase 2 Project Committees will be implemented and assume that reporting role.
+- Responsibilities:  
+  - Code Review and Quality Assurance: Ensuring that all code contributions meet the project's standards and adhere to PRAGMA policies.
```

Comment:
Do we have policies for quality and code reviews? I don’t think we do, or should, since that could interfere with project independence.